mir-module/src/main/resources/META-INF/resources/js/mir/mycore2orcid.js

```diff
@@ -1,73 +1,98 @@
-var orcidStatusURL  = webApplicationBaseURL + "rsc/orcid/status/";
-var orcidPublishURL = webApplicationBaseURL + "rsc/orcid/publish/";
+(function(){
+	const orcidStatusURL  = window["webApplicationBaseURL"] + "rsc/orcid/status/";
+	const orcidPublishURL = window["webApplicationBaseURL"] + "rsc/orcid/publish/";
+	const translateURL = window["webApplicationBaseURL"] + "rsc/locale/translate/";
 
-var orcidIcon = "<img alt='ORCID iD' src='" + webApplicationBaseURL + "images/orcid_icon.svg' class='orcid-icon' />";
+	const orcidIcon = "<img alt='ORCID iD' src='" + webApplicationBaseURL + "images/orcid_icon.svg' class='orcid-icon' />";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['webApplicationBaseURL' is not defined.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=5519111)